Libplanet.Store/Trie/ITrie.cs

```diff
@@ -59,5 +70,31 @@ public interface ITrie
         /// values are ordered in the same way to the corresponding <paramref name="keys"/>.  Absent
         /// values are represented as <see langword="null"/>.</returns>
         IReadOnlyList<IValue?> Get(IReadOnlyList<KeyBytes> keys);
+
+        /// <summary>
+        /// Gets the first node encountered at <paramref name="nibbles"/> when traversing down
+        /// from <see cref="Root"/>.
+        /// </summary>
+        /// <param name="nibbles">The path to check.  This must be a secure path.</param>
```

Comment:
> This must be a secure path.

I wonder that this `secure` word has same meaning with `TrieStateStore.Secure` (hash path and use the hash as path)

Reply:
Yes, it is the same `secure`. Probably could be worded better. 🙄